runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -672,11 +680,11 @@ public NettyClientRouter(String host, Integer port, Boolean tls,
     public NettyClientRouter(@Nonnull NodeLocator node,
         @Nonnull CorfuRuntimeParameters parameters) {
         this(node, parameters.getSocketType()
-            .getGenerator().generate(Runtime.getRuntime().availableProcessors() * 2,
+            .getGenerator().generate(parameters.getNettyEventLoopThreads(),
                 new ThreadFactoryBuilder()
                     .setDaemon(true)
                     .setNameFormat(parameters.getNettyEventLoopThreadFormat())
-                    .build()), parameters);
+                    .build()), parameters, true);
     }
```

Comment:
Can we remove the `manageEventLoop` flag, and set the `eventLoopGroup` field in this contructor only? Then in close() we can simply check if `eventLoopGroup` is null and close it it not null.

Reply:
Discussed offline. This change is less invasive.